system-int-test/test/cmr/system_int_test/search/collection_concept_retrieval_test.clj

```diff
@@ -383,16 +383,16 @@
         ;; to vars to make it easier to see what is being ingested.
 
         ;; Ingest a collection twice.
-        coll1-1 (d/ingest "PROV1" umm-coll1-1)
-        coll1-2 (d/ingest "PROV1" umm-coll1-2)
+        _coll1-1 (d/ingest "PROV1" umm-coll1-1)
```

Comment:
Cases where there are a slew of initializations done in a let block but where we don't actually care about the returned object could be moved to outside the let.

Reply:
There is a note above that says `NOTE - most of the following bindings could be ignored with _, but they are assigned to vars to make it easier to see what is being ingested.` So I think we can keep it in there for easier reading